cclib/parser/logfileparser.py

```diff
@@ -434,19 +437,28 @@ def new_internal_job(self):
             if hasattr(self, name):
                 delattr(self, name)
 
-    def set_attribute(self, name, value, check=True):
-        """Set an attribute and perform a check when it already exists.
+    def set_attribute(self, name, value, check_change=True):
+        """Set an attribute and perform an optional check when it already exists.
 
         Note that this can be used for scalars and lists alike, whenever we want
-        to set a value for an attribute. By default we want to check that
-        the value does not change if the attribute already exists, and this function
-        is a good place to add more tests in the future.
+        to set a value for an attribute.
+        
+        Parameters
+        ----------
+        name: str
+            The name of the attribute.
+        value: str
+            The value for the attribute.
+        check_change: bool
+            By default we want to check that the value does not change
+            if the attribute already exists.
```

Comment:
Are we going to stick to NumPy-style docstrings? If we pick something, it should go in the developer docs.

Reply:
Added to #555. I'm not convinced by any particular format, but I'm OK with NumPy.

Reply:
I've written each of those forms, I have no preference.